client/components/styles/Menu/item.js

```diff
@@ -0,0 +1,15 @@
+import styled from 'styled-components';
+
+export default styled.div`
+  width: 140px;
+  margin: auto;
+  text-alignt: left;
+  padding: 16px;
+  font-size: 13px;
+  cursor: pointer;
+  color: ${(props) => (props.current === props.tab ? '#20cd99' : 'white')}
```

Comment:
This is cool. What does it do?

Reply:
Styled components can take it props. So for this drop down menu, I want the color to be highlighted if its clicked on. So when an item is clicked, I pass in the clicked item and then which tab the user is currently on. If the tab and clicked item match, then the color of the text is highlighted